identifier/backends/ldap/ldap.go

```diff
@@ -387,8 +390,15 @@ func (b *LDAPIdentifierBackend) Logon(ctx context.Context, audience, username, p
 		return false, nil, nil, nil, fmt.Errorf("ldap identifier backend logon search returned wrong user")
 	}
 
-	// Bind as the user to verify the password.
-	err = l.Bind(entry.DN, password)
+	// use a temporary connection for the user Bind to verify the password.
```

Comment:
Please use the same comment style as the existing comments (start with uppercase) - there are more similarly wrong comments below.